test/solidity-ibc/IBCAdminTest.t.sol

```diff
@@ -142,4 +151,52 @@ contract IBCUUPSUpgradeableTest is Test {
         vm.expectRevert(abi.encodeWithSelector(IIBCUUPSUpgradeableErrors.Unauthorized.selector));
         ics26Router.setTimelockedAdmin(newTimelockedAdmin);
     }
+
+    function test_success_pauseAndUnpause() public {
+        assertEq(ics20Transfer.getPauser(), ics20Pauser);
+
+        vm.prank(ics20Pauser);
+        ics20Transfer.pause();
+        assert(ics20Transfer.paused());
+
+        // Try to call a paused function
+        IICS20TransferMsgs.SendTransferMsg memory sendMsg;
+        vm.expectRevert(abi.encodeWithSelector(PausableUpgradeable.EnforcedPause.selector));
+        ics20Transfer.sendTransfer(sendMsg);
+
+        vm.prank(ics20Pauser);
+        ics20Transfer.unpause();
+        assert(!ics20Transfer.paused());
```

Comment:
should probably test that the function can be called after unpause

Reply:
Probably not needed as this requires adding unrelated transfer helpers such as an erc20 contract into this test suite. I think simply checking that the contract is not paused and trusting openzeppelin is not so bad. Moreover, this can be fixed by a migration if it ever comes to it,